spikingjelly/activation_based/model/spike_driven_transformer.py

```
@@ -0,0 +1,279 @@
+    def __init__(
+        self,
+        in_channels: int,
+        embed_dims: int,
+        pooling_stat: str,
+        backend: str,
+    ):
+        super().__init__()
+        channels = (embed_dims // 8, embed_dims // 4, embed_dims // 2, embed_dims)
+        self.stages = nn.ModuleList()
+        previous = in_channels
+        for use_pool, channels_out in zip(pooling_stat, channels):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)